docs/automated_testing.md

```diff
@@ -0,0 +1,509 @@
+---
+id: automated_testing
+title: Automated Integrated Testing
+sidebar_label: Automated Testing
+slug: /automated_integrated_testing
+---
+
+<!-- original: https://docs.google.com/document/d/1hkNr78mrU3Ha98tPUL4OKWi3eNnt-1Sba7L8470u06g/edit# -->
+
+🚧 Reference for writing automated integrated tests with the Assembly Line testing framework.
+
+<!-- ## Setup
+
+For now, talk to us about it.
+
+- Have admin permissions on the repo
+- Use the setup interview (https://apps-dev.suffolklitlab.org/start/test-setup/)
+- Add this element to the page somewhere: <div data-variable="${ encode_name(str( user_info().variable )) }" id="trigger" aria-hidden="true" style="display: none;"></div>
+
+Secrets
+
+PLAYGROUND_PASSWORD
+PLAYGROUND_ID
+PLAYGROUND_EMAIL
+-->
+
+## Summary
+
+1. You write and edit `.feature` test files in your Sources folder.
+1. There's a maximum of 2 minutes per individual Step or table row. Longer than that and the test will error. That can't be customized yet.
+1. Tests are run when you commit.
```

Comment:
```suggestion
1. Tests are run when you commit (there's a maximum of 2 minutes per individual Step or table row. Longer than that and the test will error. That can't be customized yet).
```

Reply:
I'd like each item to be as short and well-contained as possible. I think the two concepts are distinct and unrelated. I only mention the timeout because I think it's pretty crucial. How about just switching the order?

```suggestion
1. Tests are run when you commit.
1. There's a maximum of 2 minutes per individual Step or table row. Longer than that and the test will error. That can't be customized yet.
```

Reply:
Taking a step back I guess, what are these items? They read more like steps that the test author takes, but the "There's a maximum of 2 minutes per individual Step..." item isn't something you can do. The second item probably doesn't need to be mentioned here, and can be moved later, even if it's crucial (i.e., is it information that someone, encountering the testing system for the first time, would even remember by the time they got to writing tests?)

Reply:
Ah, I see. Can we leave it this way for the MVP and revisit the best place to put this info when re-organizing a la divio? If so, resolve for now and we can make an issue for it to avoid forgetting.

Reply:
#138